kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWritesSuite.scala

```diff
@@ -1256,4 +1256,33 @@ class DeltaTableWritesSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBa
       assert(meta.get(ColumnMapping.COLUMN_MAPPING_PHYSICAL_NAME_KEY) == expPhyName)
     }
   }
+
+  test("ensure column mapping is disabled - for Delta 3.3 only") {
+    Seq(
+      (TableConfig.COLUMN_MAPPING_MODE.getKey, "id"),
+      (TableConfig.ICEBERG_COMPAT_V2_ENABLED.getKey, "true")
+    ).foreach { case (key, value) =>
+      withTempDirAndEngine { (tablePath, engine) =>
+        val ex = intercept[UnknownConfigurationException] {
+          createTxn(engine, tablePath, isNewTable = true, testSchema, partCols = Seq.empty,
+            tableProperties = Map(key -> value))
+            .commit(engine, emptyIterable())
+        }
+        assert(ex.getMessage.contains("Unknown configuration was specified:"))
+      }
+    }
+  }
+
+  // helper method to run columnmapping tests
+  def withColumnMappingEnabled(testName: String)(fun: => Unit): Unit = {
```

Comment:
maybe this should be with more specific to column mapping _enablement_ being enabled? confusing wording :(

Reply:
renamed.